src/core_modules/capture-core-utils/date/date.utils.js

```diff
@@ -67,8 +75,3 @@ export const displayTime = (() => {
         return date.format((displayType && typeToMomentConverter[displayType]) || typeToMomentConverter[displayTypes.MEDIUM]);
     };
 })();
-
-export const RulesEngineDateUtils = {
```

Comment:
what makes us wanna delete this bit?

Reply:
Agreed, deleted them all. The reason the RulesEngineDateUtils was deleted in the first place  was because it was incorrect.